grommet/hpe-design-system
# DRAFT - October What's New

#### What does this PR do?

- Drafts "what's new" content thus far in October. [Preview](https://deploy-preview-1987--keen-mayer-a86c8b.netlify.app/whats-new)
- Experiments with providing a bit more of a narrative for readers, but still in a bulleted format. Narrative attempts to focus on what is relevant for this audience:
    - New features, enhancements, or bug fixes related to requests received from Design System consumers.
    - Any new or revised guidance for which folks should be aware
    - Progress updates on **Big Things** initiatives to provide a bit more visibility
    - Adjustments to any communication formats and or venues

#### Where should the reviewer start?

[What's New page](https://deploy-preview-1987--keen-mayer-a86c8b.netlify.app/whats-new)

#### What testing has been done on this PR?

#### How should this be manually tested?

- Proof read
- Check links
- Check for any missing notable updates
- Reword, correct, clarify

#### Any background context you want to provide?

#### What are the relevant issues?

#### Screenshots (if appropriate)

#### Should this PR be mentioned in Design System updates?

#### Is this change backwards compatible or is it a breaking change?
